logging/airgap/airgap-fluent-bit.yaml

```diff
@@ -0,0 +1,5 @@
+image:
```

Comment:
I don't think we need to "airgap-" in the names of these files since they are in the "airgap" directory.